### ISSUE TITLE
cy.viewport doesn't take effect if `width` not changed

### ISSUE THREAD

**Issue by dwelle** [cypress-io/cypress#873]
- Operating System: Win7x64
- Cypress Version: 1.0.3

### Is this a Feature or Bug?

Bug

### Current behavior:

Issuing `cy.viewport( 1000, 200 )` doesn't change the viewport size if current viewport size is `1000x600`, and thus only `height` is being changed (to `200`).

### Desired behavior:

Should change viewport height to `200`

**Comment by jennifer-shehane** [comment]
I'm not sure that I understand your issue exactly. I recreated this, by setting by default viewport to `1000x600` and issuing a `cy.viewport( 1000, 200 )`. The viewport size changes to `1000` width and `200` height during the test run. The viewport then changes back to the default viewport size of `1000x600` after the test is finished. You should be able to hover over or click on the commands during the test run and see the viewport at size `1000 x 200`. https://docs.cypress.io/guides/core-concepts/overview-of-test-runner.html#Command-Log

**Comment by dwelle** [comment]
Doesn't do anything when the `width` is already 1000. From the [source](https://github.com/cypress-io/cypress/blob/develop/packages/driver/src/cy/commands/window.coffee#L49) it seems the reason is because the height isn't checked correctly:

```
if currentViewport.viewportWidth is width and currentViewport.viewportHeight
```

Thus, it shouldn't work on your end, too :thinking:

**Comment by jennifer-shehane** [comment]
Link to code: https://github.com/cypress-io/cypress/blob/code-of-conduct/packages/driver/src/cy/commands/window.coffee#L49

This code does look like it may be a problem. A failing test on our end would be helpful in confirming the behavior so it can be fixed.

**Comment by chrisbreiding** [comment]
The code for this is done, but has yet to be released. We'll update this issue and reference the changelog when it's released.

**Comment by brian-mann** [comment]
Released in [`1.1.0`](https://docs.cypress.io/guides/references/changelog.html#1-1-0).
